deploy/modules/gcp/tidb-cluster/main.tf

```diff
@@ -145,6 +145,7 @@ module "tidb-cluster" {
   kubeconfig_filename        = var.kubeconfig_path
   base_values                = file("${path.module}/values/default.yaml")
   wait_on_resource           = [google_container_node_pool.tidb_pool, var.tidb_operator_id]
+  service_ingress_key        = "ip"
```

Comment:
`tidb-cluster-release` is used in gcp, aliyun and aws. why this variable is only set for gcp?

Reply:
It's already set in aliyun and aws.

Reply:
ah, I see. other platforms use its default value.